Text_Mining.py

```diff
@@ -0,0 +1,161 @@
+import requests
+import pickle
+import os.path
+import nltk
+
+
+"""Downloads 'Alice in Wonderland'
+"""
+
+# Save data to a file (will be part of your data fetching script)
+if os.path.isfile('alice_in_wonderland.pickle') == False:
+    alice_in_wonderland_full_text = requests.get('http://www.gutenberg.org/files/11/11-0.txt').text
+    f = open('alice_in_wonderland.pickle', 'wb')
+    pickle.dump(alice_in_wonderland_full_text, f)
+    f.close()
+else:
+    # Load data from a file (will be part of your data processing script)
+    input_file = open('alice_in_wonderland.pickle', 'rb')
+    reloaded_copy_of_texts1 = pickle.load(input_file)
+
+
+"""Downloads 'Tom Sawyer'"""
+
+if os.path.isfile('tom_sawyer.pickle') == False:
+    Tom_Sawyer_full_text = requests.get('http://www.gutenberg.org/files/74/74-0.txt').text
+    f2 = open('tom_sawyer.pickle', 'wb')
+    pickle.dump(Tom_Sawyer_full_text, f)
+    f.close()
+else:
+    # Load data from a file (will be part of your data processing script)
+    input_file2 = open('tom_sawyer.pickle', 'rb')
+    reloaded_copy_of_texts2 = pickle.load(input_file2)
+
+"""Cuts out beginning and end sections from each book"""
```

Comment:
Usually, docstring comment appear as a first statement after function definition. Look at [official Python guideline](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring), if you are interested to learn more. More conventional way to put your comment here would be something like this:
```
def cut_alice(text):
    """Cuts out beginning and end sections from each book"""
```